fuzz/Makefile

```diff
@@ -11,7 +11,7 @@ check: ## Checks that fuzz member compiles
 	cargo check
 
 build-target: ## Build the target fuzz
-	cargo rustc --bin $(TARGET) $(ARGS) -- \
+	cargo +nightly rustc --bin $(TARGET) $(ARGS) -- \
```

Comment:
Any reason why we need to add `nightly` here? Since the root workspace expects `stable`, we might have a divergence there

Reply:
I've added nighly because it complained about `-Z sanitizer=address`

Reply:
```
TARGET=parse_address make build-target 
cargo rustc --bin parse_address  -- \
                -C debuginfo=full \
                -C debug-assertions \
                -C passes='sancov-module' \
                -C llvm-args='-sanitizer-coverage-level=3' \
                -C llvm-args='-sanitizer-coverage-inline-8bit-counters' \
                -Z sanitizer=address
   Compiling sovereign-sdk-fuzz v0.3.0 (/Users/nikolai/workspace/sovereign/sovereign-sdk/fuzz)
error: the option `Z` is only accepted on the nightly compiler

```